addons/live-comments/addon.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Realtime Comments",
+  "description": "Adds new comments and replies to comments sections in real-time",
+  "credits": [
+    {
+      "name": "MeechaPooch (@ilhp10)",
+      "link": "https://github.com/MeechaPooch",
+      "note": "addon, profile, button",
+      "id": "addon"
+    },
+    {
+      "name": "CubeyTheCube",
+      "link": "https://github.com/CubeyTheCube",
+      "note": "project & studio",
+      "id": "project"
+    },
+    {
+      "name": "Weredime (9gr)",
+      "link": "https://scratch.mit.edu/users/9gr/",
+      "note": "improvements",
+      "id": "projec"
```

Comment:
```suggestion
      "id": "project2"
```

Reply:
Did you mean just `project`?

Reply:
That is was already taken above, but maybe some of these credit notes aren't necessary?